static/app/data/platformCategories.tsx

```diff
@@ -346,13 +346,14 @@ export const replayBackendPlatforms: readonly PlatformKey[] = [
 
 // These are the frontend platforms that can set up replay.
 export const replayFrontendPlatforms: readonly PlatformKey[] = [
-  'capacitor',
-  'electron',
+  'javascript',
```

Comment:
some kind of comment like this will be helpful for the next person who comes by and wants to sort the list:
```suggestion
  'javascript', // First in the list so that it can be used as the default option in platform choosing dropdown
```

Reply:
this list actually does nothing :( what pushes `javascript` to the top is the logic i do inside `utils.tsx`

Reply:
can add a comment there though